CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1)
+
+project(threadpool)
+
+set(CMAKE_CXX_STANDARD 11)
+add_compile_options(-Wall)
```

Comment:
This does not make sense to all compilers.

Reply:
You are right, but '-Wall' does works on gcc and clang, this two platform are most widely used.
And, it just for example use.